src/bin/travis/resources/centaur.inputs

```diff
@@ -1,6 +1,7 @@
 {
   "centaur.centaur.cromwell_jar":"gs://cloud-cromwell-dev/travis-centaur/CROMWELL_JAR",
   "centaur.centaur.cromwell_branch":"BRANCH",
-  "centaur.centaur.conf":"gs://cloud-cromwell-dev/travis-centaur/jes.conf",
-  "centaur.centaur.pem":"gs://cloud-cromwell-dev/travis-centaur/cromwell-account.pem"
+  "centaur.centaur.conf":"gs://cloud-cromwell-dev/travis-centaur/multiBackend.conf",
+  "centaur.centaur.pem":"gs://cloud-cromwell-dev/travis-centaur/cromwell-account.pem",
```

Comment:
gsutil while auth'd as what project?

When I looked previously the general public couldn't see this stuff